examples/with-nextjs-and-clerk-auth/src/app/(monite)/payables/loading.tsx

```diff
@@ -0,0 +1,5 @@
+import LoadingFallback from '@/components/LoadingFallback';
+
+export default function PayablesLoading() {
+  return <LoadingFallback text="Loading payables..." />;
```

Comment:
We refer to this as Bill Pay in the demo

```suggestion
  return <LoadingFallback text="Loading Bill Pay..." />;
```